silverback/application.py

```diff
@@ -118,10 +118,18 @@ def broker_task_decorator(
 
         # Register user function as task handler with our broker
         def add_taskiq_task(handler: Callable) -> AsyncTaskiqDecoratedTask:
+            labels = {"task_type": str(task_type)}
+
+            if container and isinstance(container, ContractEvent):
+                # Address is almost a certainty if the container is being used as a filter here.
+                if contract_address := getattr(container.contract, "address", None):
+                    labels["contract_address"] = contract_address
+                labels["event_abi"] = container.abi.model_dump_json()
```

Comment:
I think this will add a lot of extra weight to the labels, would it make more sense for downstream usage to add the topic IDs instead? this way it can just feed straight into the subscription

Reply:
That's not enough information to reconstruct the contract event container.  Could maybe boil it down to the even signature and re-parse that into an `EventABI`. Is a KB or so that "heavy"?

Reply:
Currently, all the labels are being displayed in logs every time a task is executed, so yes 1KB is a little heavy

We could filter it out but I was brainstorming if there was another way to solve this

Reply:
Now using the selector for the label in ef14b8a.  Paired with [the recent change in ethpm-types](https://github.com/ApeWorX/ethpm-types/pull/121).

Reply:
can you add a lower pin on this version of ethpm-types?

Reply:
Yeah, wanted to avoid adding it as a direct dep since it's currently a peer, but I guess that makes sense.

Reply:
> Yeah, wanted to avoid adding it as a direct dep since it's currently a peer, but I guess that makes sense.

yeah, hence why only a lower pin haha

Reply:
added in 5245f53